var/spack/repos/builtin/packages/zee/package.py

```diff
@@ -48,6 +29,10 @@ class Zee(CMakePackage):
             description='Compile examples using PETSc')
     depends_on('cmake@3:', type='build')
     depends_on('pkg-config', type='build')
+    depends_on('py-cmake-format', type='build')
+    depends_on('py-pre-commit', type='build')
+    depends_on('py-pyyaml', type='build')
+    depends_on('python@3:', type='build')
```

Comment:
@tristan0x  : I wonder what is best way to add these dependencies. These are more development dependencies and not strictly required for deployment  of a package. Should there be variant for such dependencies?

Reply:
I am not against using a variant to enable those dependencies. Is there a proper variant name for such purpose in spack already?

Reply:
as you know, there is `develop` version which typically points to master. we don't expect coding converts to be applied for other released versions? in this case I wonder if we can use `develop`.

Reply:
That is a very good idea. Thanks